src/app/legacy/psammead/psammead-navigation/src/index.jsx

```diff
@@ -8,6 +8,7 @@ import {
   C_GREY_10,
   C_GREY_3,
   C_POSTBOX,
+  C_BLACK,
```

Comment:
As per my comment above - could we import `BLACK` from https://github.com/bbc/simorgh/blob/latest/src/app/components/ThemeProvider/palette.ts#L2

Reply:
Have updated 👍